README.md

```diff
@@ -149,6 +149,30 @@ You can update multiple docs versions at the same time because each directory in
 
 Example: When you change any file in `versioned_docs/version-0.7.0/`, it will only affect the docs for version `0.7.0`.
 
+## Configs
+Configs can be automatically updated by following these steps.
+
+Execute these from hudi-utils dir <br/>
+Ensure you have hudi artifacts from latest master has been installed in your local maven repo before proceeding. <br/>
+If not execute `mvn install -DskipTests` in your hudi repo  <br/>
+
+```shell
+mvn clean
+mvn install
+```
+Set the appropriate SNAPSHOT version and execute the below commands
+```shell
+VERSION=0.11.0
+```
+
+```shell
+java -cp target/hudi-utils-1.0-SNAPSHOT-jar-with-dependencies.jar:$HOME/.m2/repository/org/apache/hudi/hudi-utilities-bundle_2.11/$VERSION-SNAPSHOT/hudi-utilities-bundle_2.11-$VERSION-SNAPSHOT.jar:$HOME/.m2/repository/org/apache/hudi/hudi-spark-bundle_2.11/$VERSION-SNAPSHOT/hudi-spark-bundle_2.11-$VERSION-SNAPSHOT.jar:$HOME/.m2/repository/org/apache/hudi/hudi-flink-bundle_2.11/$VERSION-SNAPSHOT/hudi-flink-bundle_2.11-$VERSION-SNAPSHOT.jar:$HOME/.m2/repository/org/apache/hudi/hudi-kafka-connect-bundle/$VERSION-SNAPSHOT/hudi-kafka-connect-bundle-$VERSION-SNAPSHOT.jar org.apache.hudi.utils.HoodieConfigDocGenerator
```

Comment:
`hudi-utils-1.0-SNAPSHOT-jar-with-dependencies.jar`
-> is the version `1.0-SNAPSHOT` or `0.11.0-SNAPSHOT`?

Reply:
utils is local module. it is 1.0